src/Type/WPObjectType.php

```diff
@@ -127,6 +126,22 @@ public function prepare_fields( $fields, $type_name ) {
 		 */
 		$fields = apply_filters( 'graphql_object_fields', $fields, $type_name );
 
+		// Get any parent interface fields.
+		$parent_fields = [];
+		if ( ! empty( $config['interfaces'] ) ) {
```

Comment:
```suggestion
		if ( ! empty( $config['interfaces'] ) && is_array( $config['interfaces'] ) ) {
```

Reply:
Can it be anything other than an `array`

Reply:
it's not supposed to be, but we don't want to loop over it if it were a string or something else. It's a value 3rd party devs will be filling out, so we don't want to iterate over a non-array should they fill it out incorrect. 

Perhaps if it's not an array, we throw an error so they know they messed up. 🤔

Reply:
Yea, that would probably be the best bet.